doc/dev/cephfs-snapshots.rst

```diff
@@ -1,7 +1,7 @@
 CephFS Snapshots
 ================
 
-CephFS supports snapshots, generally created by invoking mkdir against the
+CephFS supports snapshots, generally created by invoking mkdir within the
 (hidden, special) .snap directory.
```

Comment:
(hidden, special, not visible in a dirlisting)

Maybe try to be as explicit as possible?